ecosystem/sep-0010.md

```diff
@@ -19,56 +19,56 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
 It involves three components:
-- A domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`), referred to as the home domain.
-- An endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on the home domain, a sub-domain of the home domain, or any other domain. The domain where the endpoint is hosted is referred to as the web auth domain. 
-- A client who will authenticate using a Stellar account.
+- A **Home Domain**: a domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` (URL) and `SIGNING_KEY` (`G...`). The ``WEB_AUTH_ENDPOINT`` must be hosted by a **Web Auth Endpoint** as described below.
+- A **Web Auth Endpoint**: a domain hosting an endpoint providing the GET and POST operations discussed in this document. The endpoint may be hosted on a **Home Domain**, a sub-domain of a **Home Domain**, or any other domain.
+- An **Authenticating Account**: The public key of the Stellar account being authenticated.
```

Comment:
❗ The component is the account not the public key. In the API we might reference the public key but the component in play is the account itself.
```suggestion
- An **Authenticating Account**: a Stellar account being authenticated.
```

Reply:
I don't feel strongly either way, but the account may not actually exist, which is why I chose to mention the public key specifically.